### PR TITLE
Added Documentation

### LINKED ISSUE #2: Can't execute `conda install` from Setup script
**Describe the bug**
Trying to use conda install does not work inside a bash script. 

**To Reproduce**
1. Ran sh Script.sh or ./Script.sh on Powershell
2.  See error - Setup.sh: 5: Setup.sh: conda: not found

**Expected behavior**
Script should run without errors

**Screenshots**
NA.

**Desktop (please complete the following information):**
 - OS: [Win 10]
 - Command-line interpreter: [Powershell]
 - Version [5.1.19041.1]

**Additional context**
NA

### LINKED ISSUE #4: python3 doesn't work in Conda
**Describe the bug**
Any of the commands starting with python3, like python3 AllDS2020.py Train do not work in Conda

**To Reproduce**
1. Follow the steps mentioned in README.md
2. While starting training in Conda environment, run python3 AllDS2020.py Train 
4. See error(s) - Nothing happens or Windows opens up the Store application

**Expected behavior**
Steps should run as expected and mentioned in the README.md file

**Screenshots**
NA

**Desktop (please complete the following information):**
 - OS: [Win 10]
 - Command-line interpreter: [Powershell]
 - Version [5.1.19041.1]

**Additional context**
NA

### PR DESCRIPTION
**Pull request by rishabhbanga**
1. Created documentation for Windows Issues but named it as Issues.md to keep it generalized.
2. Created documentation to explain functioning of scripts in Scripts folder - Autopilot feature
3. Added Values for Windows 10 in CNN's README.md

### CODE REVIEW

**Review by AdamMiltonBarker — CHANGES_REQUESTED** [review]
Hi mate sorry changes:

- Name the file Windows-10.md not issues.md
- Changing the profiles links was a mistake on my part please can you leave the association links as they are but the individuals' profiles should point to research project website as we are in the RP repo. Sorry this was my fault. 
- To keep inline with all other repos, file names and directories should begin with capital letter.
- Were we not meant to make the scripts executable by the Scripts directory, I feel moving files is unrequired  step. 
- Line breaks ( & nbsp ; ) should always be present between level 1 titles to keep in line with the rest of our repos, this isn't a mistake but intentional. 
- File names should not have spaces in them, same for directories.
- sh scriptsSetup.sh should be CamelCase

I think this is all

**Comment by rishabhbanga** [comment]
Hi,

1. I hope you read the description - Created documentation for Windows Issues but named it as Issues.md to keep it generalized. This is because in case people start facing issues in MAC and Linux it would be better to have a single source of information.
2. So change individual links like https://www.leukemiaresearchassociation.ai/team/adam-milton-barker to https://www.leukemiaresearchassociation.ai/ ? 
3. If you're talking about the shell scripts. They're always supposed to being with small letters, though just saw I left Autorun.sh with a Capital letter.
4. You never replied to that discussion citing it. I assumed you agreed with the idea of keeping some level of abstraction.
5. But does it mean we should keep it that ways? The point of removing it was to maintain some best practices to make the documentation clean and not with unnecessary gaps.
6. Not sure why you mentioned this, but I double checked and couldn't find any such gaps between the files and directories I uploaded.
7. scriptsSetup is Camel Case. Don't understand your comment here.

**Comment by rishabhbanga** [comment]
Ah I saw I uploaded the Article Ideas.md! Yeah that wasn't supposed to go up. But we could keep it there?

**Comment by AdamMiltonBarker** [comment]
Hi @rishabhbanga  it isn't required no mate.

**Comment by AdamMiltonBarker** [comment]
1. This was a fix for Windows no? Issues is not a descriptive name. If we have issues related this on other OS which we don't we will make OS specific files.
2. No the names should point to profile on https://www.petermossamlallresearch.com
3. Capital letters for all file names and directories.
4. No requirement to move files around you can run it from the Scripts directory.
5. You removed some, they need to go back to keep inline with all repositories they are by design not fault if you see other repositories it has been like this for a long time no need to remove.
6. Shouldn't be there
7. CamelCase capital C's

Thanks Rishabh

**Comment by rishabhbanga** [comment]
Pushed the changes. Javier doesn't have a profile on https://www.petermossamlallresearch.com so didn't change his.

Moreover since you mentioned only profiles Peter Moss Leukemia AI Research still points to https://www.leukemiaresearchassociation.ai

Remembered Line breaks ( & nbsp ; ) comment hasn't been integrated will push that soon

File: CNN/Scripts/ScriptsSetup.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+echo "Copying scripts to their respective directories"
+echo " "
+echo "Copying Autorun.sh, Randomizer.sh and Timediff.sh scripts to the CNN directory"
+cp Autorun.sh ../
+cp Timediff.sh ../
+cp Randomizer.sh ../Model/Data/Train/
```
Comment (AdamMiltonBarker) [inline CNN/Scripts/ScriptsSetup.sh]: The script is still moving files, this is unrequired and can be changed to run from the Scripts directory., To confirm, we do not need to move any files.

File: Documentation/AutoPilot.md [hunk 2]
```diff
@@ -1,63 +1,59 @@
...
+
+1. ScriptsSetup.sh
+2. Autorun.sh
+3. Randomizer.sh
+4. Timediff.sh
+
+All of these are located inside the Scripts folder of the CNN Directory. From here we run
+
+```
+sh ScriptsSetup.sh
```
Comment (AdamMiltonBarker) [inline Documentation/AutoPilot.md]: Include cd Scripts

File: Documentation/AutoPilot.md [hunk 3]
```diff
@@ -1,63 +1,59 @@
...
+```
+
+**ScriptsSetup.sh** transfer all the 3 scripts to their respective folders and from their internally runs 
+
+```
+sh Autorun.sh
```
Comment (AdamMiltonBarker) [inline Documentation/AutoPilot.md:22]: Include cd Scripts

**Review by AdamMiltonBarker — CHANGES_REQUESTED** [review]

**Comment by AdamMiltonBarker** [comment]
Thanks @rishabhbanga  just a few last changes that haven't been done yet then I believe we can merge your code. Thank you for the hard work.

**Pull request merged**